plugins/inputs/vsphere/vsphere.go

```diff
@@ -50,13 +50,14 @@ type VSphere struct {
 	IPAddresses             []string
 	MetricLookback          int
 
-	MaxQueryObjects         int
-	MaxQueryMetrics         int
-	CollectConcurrency      int
-	DiscoverConcurrency     int
-	ForceDiscoverOnInit     bool
-	ObjectDiscoveryInterval config.Duration
-	Timeout                 config.Duration
+	MaxQueryObjects            int
+	MaxQueryMetrics            int
+	CollectConcurrency         int
+	DiscoverConcurrency        int
+	ForceDiscoverOnInit        bool
+	ObjectDiscoveryInterval    config.Duration
+	Timeout                    config.Duration
+	HistoricalIntervalDuration int
```

Comment:
Instead of `int` please also make it a `config.Duration`. That simplifies it for all users.

Reply:
Yeah I had gone back and forth on whether it should include "Duration" at the end of the name, I think it makes sense to drop it...  
Switched to using config.Duration.  New to Go and time.Duration is....different :)